scalableminds/webknossos
# re-report datasource json on dataset reload trigger

The frontend could directly use the returned datasource to update the values in the list. Also, I did not change the frontend code so there the functions are still called clearCache (for the record, the cache is still cleared, but the call _also_ reloads the datasource properties). ~~@daniel-wer @philippotto does one of you want to take this?~~


### URL of deployed dev instance (used for testing):
- https://reloaddatasource.webknossos.xyz

### Steps to test:
- change datasource properties on disk
- hit “reload” in wk dataset list
- new datasource properties should be reported to wK

### Issues:
- fixes #3734 

------
